CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,337 @@
+# 1. Code of Conduct for the Open Research Community Building Team
+
+> _A code of conduct is a set of rules outlining the norms, rules, and responsibilities or proper practices of an individual party or an organization._
+
+This Code of Conduct document outlines our expectations for 
+participants within the team as well as this online repository (project).
+We also provide steps to reporting any concern, uncomfortable situation 
+or unacceptable behaviour by other participants. 
+We are committed to providing a welcoming, collaborative and inspiring experience 
+for all members of this team and expect our code of conduct to be honoured.
+
+## 2. Our Commitment
+
+We as members, contributors, and leaders commit to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We commit to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## 3. Restorative Practice
+
+We want to establish a Code of Conduct that focuses on creating a welcoming and safe environment for
+everyone involved.
+If a participant does not abide by the Code of Conduct, we will use the 
+“restorative practices” approach to handle the situation. Restorative Practices has 
+their roots in “restorative justice,” a way of looking at wrongdoing that focuses on 
+repairing the harm done to people and the relationships rather than on punishing offenders or wrongdoer. 
```

Comment:
```suggestion
repairing the harm done to people and the relationships rather than on punishing offenders or wrongdoer. This approach emerged from studies of and in reaction to incarceration and punishment within criminal justice systems, and has its roots in indigenous belief systems.
```
I suggest this addition for greater context & intellectual lineage, especially for those that might not be familiar with the approach.

Reply:
Slight rewording suggestion for this context:
> Although the 'restorative justice' term was in 1970 in the context of social work and justice systems, restorative practice has its roots in indigenous practices employed in cultures all over the world.